app/client/components/Clickr.js

```diff
@@ -0,0 +1,95 @@
+
+import React from 'react';
+
+export default class Clickr extends React.Component {
+
+    constructor(props) {
+      super(props);
+      this.state = {
+        count: 0
+      }
+    }
+
+    handleClickPlus1 = (e) => {
```

Comment:
in this kind of method is a good practice to compute a `nextValue` and validate that one.
here the validation rule is "max 20", so:

```
var { count } = this.state;
var nextCount = count + 1;

if (nextCount <= 20) {
  this.setState({
    count: nextCount,
  });
}
```